### DISCUSSION TITLE
Question about the demo fractional_Poisson_2d_inverse.py

### DISCUSSION THREAD

**Discussion by arielxinsu** [lululxvi/deepxde#895] [Q&A] [answered]
I got the code but there is no explanation for this code. Do you have any reference that I can refer to ? What does the problem look like in mathematics formulation?

**Comment by lululxvi** [comment, marked as answer]
See https://doi.org/10.1137/18M1229845

**Reply by arielxinsu** [reply]
Thanks for your information.
